src/ray/raylet/worker_pool.cc

```diff
@@ -1286,7 +1287,7 @@ void WorkerPool::PrestartWorkers(const TaskSpecification &task_spec,
                                  int64_t num_available_cpus) {
   // Code path of task that needs a dedicated worker.
   if ((task_spec.IsActorCreationTask() && !task_spec.DynamicWorkerOptions().empty()) ||
-      task_spec.HasRuntimeEnv()) {
+      task_spec.HasRuntimeEnv() || task_spec.GetLanguage() != ray::Language::PYTHON) {
```

Comment:
Why did we add this? (isn't it technically a regression?)

Reply:
prestart java workers with default job_config doesn't work.